evidence-based-complementary-and-alternative-medicine.csl

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
```

Comment:
Please see https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale. If this journal is English-only, please set the "default-locale" to British or US English.